modules/apps/object/object-test/src/testFunctional/tests/CreateObject.testcase

```diff
@@ -1759,6 +1759,67 @@ definition {
 		ObjectPortlet.assertDeleteEntryButtonNotPresent(entry = "Custom Object 42");
 	}
 
+	@description = "LPS-151877 - Verify that after the entry of an object definition with account restriction is saved, the account field can not be edited"
+	@priority = 4
+	test CannotEditEntryWithAccountRestriction {
+		task ("Given: Two accounts, a Custom Object and Relationship Account to Custom Object are created (Via API)") {
+			for (var accountNum : list "1,2") {
```

Comment:
There's no need to add two accounts for this test, you call leave just one Account.